### DISCUSSION TITLE
Select All button for notifications

### DISCUSSION THREAD

**Discussion by DiogoCapitao** [directus/directus#18854] [Feature Requests]
### Summary

I would like to suggest the addition of a "**Select All**" button for notifications in Directus. Currently, managing notifications individually can be time-consuming, especially when dealing with a large number of notifications. Introducing a "Select All" button would greatly enhance the user experience by simplifying the process of managing notifications.

### Motivation

The motivation behind this feature request is to enhance the usability and efficiency of managing notifications in Directus. Currently, the process of individually selecting or deselecting notifications can be time-consuming and cumbersome, particularly when dealing with a large number of notifications.

### Detailed Design

Just a simple checkbox or a button...

### Requirements List

**1.** Must-Have: "Select All" Checkbox - Implement a functional "Select All" checkbox that allows users to select or deselect all notifications at once.
Should-Have Requirements:
**2.** Should-Have: Visual Feedback - Provide clear visual feedback to indicate the selected state of the "Select All" checkbox and individual notification checkboxes.

### Drawbacks

I don't think there are any drawbacks.

### Alternatives

**Alternative Design: Batch Selection with Shift/Ctrl Key**

- Instead of a "Select All" checkbox, users could rely on existing conventions like holding Shift or Ctrl key to select multiple notifications in a batch. This approach provides flexibility but may require more manual effort when dealing with a large number of notifications.

**Impact of Not Implementing the Feature:**

- Increased Time Consumption: Without a "Select All" feature, users will need to individually select or deselect notifications, which can be time-consuming and tedious, especially when handling a substantial number of notifications.
- User Frustration: Users who are accustomed to the convenience of a "Select All" feature in other applications may experience frustration or find the notification management process in Directus less intuitive or user-friendly.

### Adoption Strategy

idk

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
